presto-main/src/main/java/com/facebook/presto/operator/ExchangeClientStatus.java

```diff
@@ -108,4 +108,29 @@ public String toString()
                 .add("pageBufferClientStatuses", pageBufferClientStatuses)
                 .toString();
     }
+
+    @Override
+    public ExchangeClientStatus mergeWith(ExchangeClientStatus other)
+    {
+        return new ExchangeClientStatus(
+                (bufferedBytes + other.bufferedBytes) / 2,
```

Comment:
Add a comment that this is safe since we know that all clients have the same buffer size, and if that changes, we should reconsider this decission